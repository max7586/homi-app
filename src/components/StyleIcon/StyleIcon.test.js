import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StyleIcon from './StyleIcon';
import { BrowserRouter } from 'react-router-dom'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'

describe('StyleIcon component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
        <StyleIcon />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });



  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<StyleIcon  styleIconMap={'Story'}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

  it('renders the UI as expected with no style', () => {
    const tree = renderer
      .create(<StyleIcon />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});



