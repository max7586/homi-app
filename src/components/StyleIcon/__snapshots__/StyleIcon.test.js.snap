// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyleIcon component renders the UI as expected 1`] = `null`;

exports[`StyleIcon component renders the UI as expected with no style 1`] = `null`;
